ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/router-ospf.yml

```diff
@@ -185,41 +185,3 @@ router_ospf:
           default_information_originate:
             metric: 50
             metric_type: 1
-
-ethernet_interfaces:
-  - name: Ethernet1
-    switchport:
-      enabled: false
-    ospf_network_point_to_point: true
-    ospf_area: 0.0.0.1
-    ospf_cost: 99
-    ospf_authentication: message-digest
-    ospf_message_digest_keys:
-      - id: 55
-        hash_algorithm: md5
-        key: "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-loopback_interfaces:
-  - name: Loopback2
-    ospf_area: 0.0.0.2
-port_channel_interfaces:
-  - name: Port-Channel12
-    switchport:
-      enabled: false
-    ospf_network_point_to_point: true
-    ospf_area: 0.0.0.12
-    ospf_cost: 99
-    ospf_authentication: message-digest
-    ospf_message_digest_keys:
-      - id: 55
-        hash_algorithm: md5
-        key: "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
we are losing ospf support for port-channel

Reply:
Added.